src/inference/include/ie/npu/myriad_config.hpp

```diff
@@ -60,7 +60,7 @@ INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_VPU_CONFIG(MYRIAD_PCIE);
 INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_VPU_CONFIG(MYRIAD_USB);
```

Comment:
Please update device name from VPU to NPU.

Reply:
old headers should be untouched

Reply:
Done.